lib/svg-sprite/utils/index.js

```diff
@@ -41,9 +41,19 @@ function isString(value) {
     return Object.prototype.toString.call(value) === '[object String]';
 }
 
+/**
+ * @param {Array} array1    First array
+ * @param {Array} array2    Second array
+ * @returns {object}        The zipped Object
+ */
+function zipObject(array1, array2) {
```

Comment:
I'm thinking this arguments should be checked to be an array. Instead throw error.

Reply:
I thought about this too, but didn't add this yet. Feel free to push :)